server/src/main/java/org/cloudfoundry/identity/uaa/oauth/token/Saml2TokenGranter.java

```diff
@@ -0,0 +1,500 @@
+/*******************************************************************************
+ *     Cloud Foundry
+ *     Copyright (c) [2009-2016] SAP SE. All Rights Reserved.
+ *
+ *     This product is licensed to you under the Apache License, Version 2.0 (the "License").
+ *     You may not use this product except in compliance with the License.
+ *
+ *     This product includes a number of subcomponents with
+ *     separate copyright notices and license terms. Your use of these
+ *     subcomponents is subject to the terms and conditions of the
+ *     subcomponent's license, as noted in the LICENSE file.
+ *******************************************************************************/
```

Comment:
I simply took it from our templates. If not OK, you can change it. It should only be clear that it comes from SAP ;)